fw/http_tbl.c

```diff
@@ -491,24 +491,36 @@ tfw_cfgop_http_rule(TfwCfgSpec *cs, TfwCfgEntry *e)
 		}
 		rule->act.type = TFW_HTTP_MATCH_ACT_MARK;
 	}
-	else if (strlen(action) && action[0] == '$') {
-		if (!strcasecmp(action, "$cache")) {
-			tfw_cfg_parse_uint(action_val, &act_val_parsed);
-			if (act_val_parsed != 1 && act_val_parsed != 0) {
-				T_ERR_NL("http_tbl: '$cache' action value "
-					 "must 0 or 1: '%s'\n",
-					 action_val);
-				return -EINVAL;
-			}
-			rule->act.type = TFW_HTTP_MATCH_ACT_FLAG;
-			rule->act.flg.set = (act_val_parsed == 0);
-			rule->act.flg.fid = TFW_HTTP_B_CHAIN_NO_CACHE;
+	else if (!strcasecmp(action, "cache_disable")) {
+		rule->act.type = TFW_HTTP_MATCH_ACT_FLAG;
+		rule->act.flg.fid = TFW_HTTP_B_CHAIN_NO_CACHE;
+		if (action_val && *action_val) {
+			rule->act.flg.set = action_val[0] == '1';
 		} else {
-			T_ERR_NL("http_tbl: only '$cache' flag setting action "
-				 "supported for now: '%s'\n",
-				 action);
+			rule->act.flg.set = 1;
+		}
```

Comment:
Now we have only global cache TTL setting and HTTP tables rules to disable caching. However, it seems for WordPress we need to be able to set custom TTLs, e.g. for `knowledge-book` vs blog comments vs the rest of almost static content. @pale-emperor could you please also check this comment in sense of https://github.com/tempesta-tech/tempesta-tech.com/issues/64 and https://github.com/tempesta-tech/tempesta-tech.com/issues/68 ?

Reply:
I already tested last commit and configure tempesta-fw as:

`cache_ttl 3600;` - on top level

And specify cache_ttl on http tables like
```
http_chain {        
	uri == "/blog/*" -> cache_ttl = 1;
        uri == "/knowledge-base/*" -> cache_ttl = 600;
       -> default;
}
```

And it works properly as expected. So i suppose everything works fine